### PR TITLE
Updated native menus

### PR DESCRIPTION
**Pull request by marionleborgne**
Updated URLs:
* HTM Studio: http://numenta.com/htm-studio/
* EULA Terms: http://numenta.com/htm-studio/?terms=1
* Feedback Form: http://numenta.com/htm-studio/#feedback
* FAQ anchor: http://numenta.com/htm-studio/#faq

Other changes:
* Pull version number from `app/package.json`
* Updated behavior of "Learn More" submenu (points to http://numenta.com/htm-studio/)
* Removed "File" menu which did not work
* Versioning documentation

### CODE REVIEW

File: unicorn/app/browser/about.html [hunk 1]
```diff
@@ -1,39 +1,58 @@
...
+</div>
+
+<script type="text/javascript">
+    var xhr = new XMLHttpRequest();
+    xhr.addEventListener('load', function () {
+        var data = JSON.parse(this.responseText);
+        document.getElementById('version').textContent = data.version;
+    });
+    xhr.open('GET', './../package.json');
+    xhr.send();
```
Comment (lscheinkman) [inline unicorn/app/browser/about.html]: Why use `XHR` here?
You could use `require` to load `package.json`
```javascript
      var package_json = require('../package.json');
      document.getElementById('version').textContent = package_json.version
```

Comment (mrcslws) [inline unicorn/app/browser/about.html]: Will that work? This isn't compiled JavaScript. Where do we import "require" from?

File: unicorn/README.md [hunk 2]
```diff
@@ -472,3 +472,11 @@ does [not yet work with Electron](https://github.com/atom/electron/issues/915).
 ### Other
 
 * [Electron and Windows Debug Symbol Server](http://electron.atom.io/docs/v0.31.0/development/setting-up-symbol-server/)
+
+# Release management
+
+## How to create a release?
+
+* Create a branch called "htm-studio-vX" where X is the version number
```
Comment (lscheinkman) [inline unicorn/README.md]: Need to update `app/package.json` with new version number

File: unicorn/app/browser/about.html (hunk 1 above)
Comment (lscheinkman) [inline unicorn/app/browser/about.html]: Just tested. It works!

Comment (lscheinkman) [inline unicorn/app/browser/about.html]: `require` comes from `electron`

Comment (mrcslws) [inline unicorn/app/browser/about.html]: ha, good catch. I'm used to web browsers.

**Pull request merged**